opensslCertGen.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+CLIENT_USAGE="Generate a self-signed client cert and key to be used in mutual TLS.\n${0} client [-o <output name>]"
+ROOT_USAGE="Generate CA root key and certificate which can only be used to sign leaf cert and not other CAs.\n${0} root [-o <output name>]"
+SERVER_USAGE="Generate a server cert key and certificate.\n${0} root -n <common name> [-o <output name>] [-r <root key if don't want it self-signed>] [-a <subjectAltName>]...\n\nExample:\n${0} server -o servertls -r rootCA.key notary-server DNS:notaryserver DNS:notary_server IP:127.0.0.1"
```

Comment:
Is `output name` a directory?.

Example doesn't contain `-a` and should the ellipsis be inside the `[-a ]` to indicate multiple values passed to that flag?